hydra-data/src/main/java/com/addthis/hydra/data/util/KeyTopper.java

```diff
@@ -63,6 +63,15 @@ public KeyTopper() {
     @FieldConfig(codable = true)
     private boolean lossy;
 
+    /**
+     * Error estimates are only supported in the BytesCodable
+     * serialization format. They are not supported
+     * in the older serialization format to preserve
+     * serialization compatibility.
+     */
+    @FieldConfig(codable = false)
```

Comment:
I believe it would suffice to omit the annotation from a programmatic perspective. I am inclined to think the javadoc by itself would also be easier to understand since it does not require a mental diff on the annotation lines.